templates/test/running_table.html.ep

```diff
@@ -6,11 +6,11 @@
 	    { targets: 0,
               className: "name" },
             { targets: 3,
+              className: "time",
               "render": function ( data, type, row ) {
-                  console.debug(new Date(data));
-		  if (type === 'display')
-		      return jQuery.timeago(new Date(data + " UTC"));
-		  else
+                    if (type === 'display') {
+		      return jQuery.timeago(new Date(data));
+                  } else
 		      return data;
```

Comment:
Ah, ok. The reason for question was that lower on line 57 there is timedate() called unconditionally so I was wondering if it is always safe to do so.

LGTM, rebase needed however.